### ISSUE TITLE
Error

### ISSUE THREAD

**Issue by BlackNine924** [WebTheDev/TicketBot#26]
When I run the bot, this error pops up
![Screenshot_2024-06-25-16-18-49-716_com android chrome](https://github.com/WebTheDev/TicketBot/assets/76858164/2f040ffd-b8ca-4e2a-80d3-bde60ca90411)

**Comment by powderedLamb40** [comment]
Have you edited the ID in the config?

**Comment by phonex-store** [comment]
I have that error too

**Comment by WebTheDev** [comment]
Hi there, most NoneType errors are config related. Please make sure you have your config.py file set appropriately.

**Comment by phonex-store** [comment]
nah, It worked the whole time and suddenly it doesn't work anymore. I think it's the code in general. Mb you can fix the error or look what the issue is?

**Comment by WebTheDev** [comment]
Hello again,
Let me see if I can replicate it and I'll reach back out with a solution.
